style.css

```diff
@@ -0,0 +1,268 @@
+.body {
+   margin: 0;
+ }
+
+ .header {
+   display: flex;
+   align-items: center;
+ }
+
+ .header::before {
+   content: '';
+   position: absolute;
+   width: 461px;
+   height: 1800px;
+   background: #000;
+ }
+
+ .header__title {
+   position: relative;
+   width: 98px;
+   height: 37px;
+   margin: 36px 0 0 52px;
+   font-family: Montserrat;
+   font-style: normal;
+   font-weight: 500;
+   font-size: 30px;
+   line-height: 37px;
+   text-align: center;
+   letter-spacing: .07em;
+   color: #fff;
+ }
+
+ .header__language {
+   position: relative;
+   display: flex;
+   width: 238px;
+   justify-content: flex-end;
+   align-items: center;
+   top: 15px;
+ }
+
+ .header__language-ru {
+   position: relative;
+   width: 31px;
+   height: 25px;
+   font-family: Montserrat;
+   font-style: normal;
+   font-weight: 500;
+   font-size: 20px;
+   line-height: 24px;
+   color: #FFF;
+   cursor: pointer;
+ }
+
+ .header__language-ru:: after {
+   content: "";
+   position: absolute;
+   background-image: url(./img/Vector.png);
+   height: 9px;
+   width: 13px;
+   left: 40px;
+   top: 9px;
+ }
+
+ .header__language-ru: hover {
+   transform: scale(1.3);
+ }
+
+ .header__menu {
+   display: flex;
+   position: relative;
+   align-self: flex-end;
+   width: 70px;
+   height: 70px;
+   background: #f5f5f3;
+   cursor: pointer;
+ }
+
+ .header__menu: hover {
+   background: #696969;
+ }
+
+ .menu__item {
+   list-style-type: none;
+   padding: 0;
+   margin: auto;
+ }
+
+ .menu__line {
+   display: flex;
+   width: 26px;
+   height: 3px;
+   background: #000;
+ }
+
+ @media (min-width: 992px) {
+   .header__menu {
+     min-height: 92px;
+     min-width: 92px;
+     margin-left: auto;
+   }
+ }
+
+   .menu__link {
+     color: navy;
+     font-size: 25px;
+   }
+
+   .icon__line:nth-child(1) {
+     top: 0px;
+   }
+
+   .icon__line:nth-child(2),
+   .icon__line:nth-child(3) {
+     top: 13px;
+   }
+
+   .icon__line:nth-child(4) {
+     top: 26px;
+   }
+
+   .menu__line:first-of-type {
+     margin-bottom: 5px;
+   }
+
+   .menu__line {
+     width: 35px;
+     height: 4px;
+   }
+
+   @media(max-width:900px) {
+     .header__menu {
+       width: 100%;
+       display: none;
+     }
+   }
+
+   .new-collection {
+     position: relative;
+     display: flex;
+     min-height: 850px;
+   }
+
+   .new-collection__girl {
+     width: 549px;
+     height: 562px;
+     margin: 86px 54px 0px 180px;
+   }
+
+   .new-collection__section {
+     position: relative;
+     top: 100px;
+   }
+
+   .new-collection__title {
+     width: 475px;
+     height: 164px;
+     margin-bottom: 36px;
+     left: 789px;
+     font-family: Montserrat;
+     font-style: normal;
+     font-weight: bold;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5986341)